http/src/main/scala/org/http4s/blaze/http/http2/SessionFlowControlImpl.scala

```diff
@@ -34,7 +34,7 @@ private class SessionFlowControlImpl(
   private[this] var _sessionUnconsumedInbound: Int = 0
 
   // exposed for testing
-  protected def onSessonBytesConsumed(consumed: Int): Unit = {
+  protected def onSessionBytesConsumed(consumed: Int): Unit = {
```

Comment:
This definitely shouldn't be changed in that way. Only via a depreciation of the current API

Reply:
I'll drop this from the PR. If the project is interested in deprecating, it should be in a smaller more focussed PR and not one w/ many other changes.